backend/src/activitypub/ActivityPub.js

```diff
@@ -33,7 +32,7 @@ export default class ActivityPub {
     if (!activityPub) {
       dotenv.config()
       const url = new URL(process.env.GRAPHQL_URI)
-      activityPub = new ActivityPub(url.hostname || 'localhost', url.port || 4000, url.origin)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-03-25T13:47:15Z" title="Monday, March 25th 2019, 2:47:15 pm +01:00">Mar 25, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -33,7 +33,9 @@ export default class ActivityPub {
     if (!activityPub) {
       dotenv.config()
       const url = new URL(process.env.GRAPHQL_URI)
-      activityPub = new ActivityPub(url.hostname || 'localhost', url.port || 4000, url.origin)
+      // TODO Check why the hostname attribute in the prod env not containing the tld! Following line is a quick fix!!
```


@mastercuber I would like to fix this *the clean way:tm:*

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-03-25T13:56:22Z" title="Monday, March 25th 2019, 2:56:22 pm +01:00">Mar 25, 2019</time>_

---


https://nodejs.org/api/url.html#url_url_host this seems to be the clean fix.